tests/pr_test.go

```diff
@@ -81,49 +50,27 @@ func TestRunFSCloudExample(t *testing.T) {
 			"region":                     region,
 			"existing_kms_instance_guid": permanentResources["hpcs_south"],
 			"kms_key_crn":                permanentResources["hpcs_south_root_key_crn"],
+			"mongodb_version":            "4.4",
```

Comment:
perhaps leave a code comment saying this should always lock into the latest supported version